pkg/crds/upgrade.go

```diff
@@ -57,7 +57,7 @@ func New(namespace string) (*Upgrader, error) {
 }
 
 // Upgrade installs the missing CRDs or updates them if they exists already
-func (u *Upgrader) Upgrade(targetVersion string) ([]unstructured.Unstructured, error) {
+func (u *Upgrader) Upgrade(chartName string, targetVersion string) ([]unstructured.Unstructured, error) {
 	extractDir, err := helmutil.GetChartTargetDir(targetVersion)
```

Comment:
This one still has hardcoded path. So it's just the targetVersion, missing chartDir.